x86_64.c

```diff
@@ -6397,6 +6397,11 @@ x86_64_ORC_init(void)
 		NULL
 	};
 	struct ORC_data *orc;
+        unsigned char orc_header[20];
+        const uint8_t orc_hash_6_4[20] = {
```

Comment:
The orc_header in hex is hard coded, according to the commit log, is a 20-byte hash. Though I have the same value in my kernel, what if the hash changed in the future, do we still need to hard code those by then? Or will the hash stay unchanged all the time?

Reply:
In addition, I would suggest to add code comment of how to get those hex number. E.g. I get those number on my machine by "readelf -x .orc_header vmlinux"